symplyphysics/laws/kinematic/vector/acceleration_of_rotating_body.py

```diff
@@ -0,0 +1,45 @@
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+)
+
+# Description
+## The acceleration of a rotating body is composed of two parts: radial acceleration, which is always
+## present in a rotating environment and which points to the center of rotation, and tangential acceleration,
+## which is responsible for the change of the magnitude of the velocity vector.
+
+# Law: a = a_r + a_t
+## a - total acceleration vector
+## a_r - radial acceleration vector (parallel to position vector of the rotating point)
```

Comment:
Probably worth mention that it is opposite to the position vector.

Reply:
It is, will do